docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+demo-superuser@cqlsh> USE test;
+demo-superuser@cqlsh:test> CREATE TABLE users (email text primary key, name text, state text);
+demo-superuser@cqlsh:test> insert into users (email, name, state) values ('john@gamil.com', 'John Smith', 'NC');
+demo-superuser@cqlsh:test> insert into users (email, name, state) values ('joe@gamil.com', 'Joe Jones', 'VA');
+demo-superuser@cqlsh:test> insert into users (email, name, state) values ('sue@help.com', 'Sue Sas', 'CA');
+demo-superuser@cqlsh:test> insert into users (email, name, state) values ('tom@yes.com', 'Tom and Jerry', 'NV');
+demo-superuser@cqlsh:test> select * from users;
+
+ email          | name          | state
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+demo-superuser@cqlsh:test> exit
+cassandra@k8ssandra-1-worker2:/$ exit
+exit
+```
+
+Try another cqlsh operation on a different cluster.
+
+
+```bash
+kubectx kind-k8ssandra-3
+
+kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
+```
+
+**Output:**
+
+```bash
+Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
+cassandra@k8ssandra-3-worker3:/$ cqlsh -u demo-superuser -p KT-ROFfbD-O9BzWS3Lxq
+Connected to demo at 127.0.0.1:9042
+[cqlsh 6.0.0 | Cassandra 4.0.1 | CQL spec 3.4.5 | Native protocol v5]
+Use HELP for help.
+demo-superuser@cqlsh> USE test;
+demo-superuser@cqlsh:test> SELECT * FROM users;
+
+ email          | name          | state
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+demo-superuser@cqlsh:test> exit
+cassandra@k8ssandra-3-worker3:/$ exit
+exit
+```
+
+Now try using the Stargate API. 
+
+**Output plus cqlsh &amp; stargate-service example:**
+
+```bash
+kubectx kind-k8ssandra-3
+
+kubectl exec --stdin --tty demo-dc3-default-sts-0 -n k8ssandra-operator -- /bin/bash
+
+Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
+cassandra@k8ssandra-3-worker3:/$ cqlsh -u demo-superuser -p KT-ROFfbD-O9BzWS3Lxq demo-dc3-stargate-service
+Connected to demo at demo-dc3-stargate-service:9042
+[cqlsh 6.0.0 | Cassandra 4.0.1 | CQL spec 3.4.5 | Native protocol v4]
+Use HELP for help.
+demo-superuser@cqlsh> use test;
+demo-superuser@cqlsh:test> select * from users;
+
+ email          | name          | state
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+```
+
+## Helm
```

Comment:
So this helm portion here is what I mentioned above as probably wanting to have.. so we just need to figure out where we want it to go, but this is what we need.